.travis.yml

```diff
@@ -144,32 +143,22 @@ install:
       elif [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
         MINCONDA_OS=$MINCONDA_OSX;
       fi
-    - wget "http://repo.continuum.io/miniconda/Miniconda3-$MINCONDA_VERSION-$MINCONDA_OS.sh" -O miniconda.sh;
+    - wget --tries=10 "http://repo.continuum.io/miniconda/Miniconda3-$MINCONDA_VERSION-$MINCONDA_OS.sh" -O miniconda.sh;
     - bash miniconda.sh -b -p $HOME/miniconda
     - export PATH="$HOME/miniconda/bin:$PATH"
     - hash -r
     - conda update -yq conda
+    - conda install -y cmake -c conda-forge
+    - conda install -y xtl==0.6.7 nlohmann_json -c conda-forge
+    - conda install -y xtensor=0.20.10 -c conda-forge
```

Comment:
Any reason why you use three separate commands here? I'd prefer just one command.

Reply:
Really sorry about this long delay - I did not receive any notif :( and sadly I got distract but Im back to finish this push!

Done